### ISSUE TITLE
"rxjs-no-unsafe-scope" false positve 

### ISSUE THREAD

**Issue by dmytro-gokun** [cartant/rxjs-tslint-rules#101]
"rxjs-no-unsafe-scope" produces a false positive for the following piece of code:

```
platformBrowserDynamic()
    .bootstrapModule(AppModule)
    .catch(err => console.log(`ERROR CAUGHT: ${err}`)); // <=== HERE
```

What is interesting to note is that "bootstrapModule" returns a Promise, not Observable. So, i do not expect "rxjs-tslint-rules" to kick in here at all.

**Comment by cartant** [comment]
> So, i do not expect "rxjs-tslint-rules" to kick in here at all.

See https://github.com/cartant/rxjs-tslint-rules/issues/56

**Comment by cartant** [comment]
Regarding #56, I don't plan on prioritising this at all. I have other projects - that will be more beneficial for RxJS developers - that need my time. I can add `console` to the whitelist, but I don't anticipate looking at #56 anytime soon.

You might want to take that into consideration when weighing up whether or not you want - or even really need - this rule.

**Comment by dmytro-gokun** [comment]
@cartant Okay, i get you. So, is it fair to say that this project will phase out (perhaps, in favor of eslint or something else)?

**Comment by cartant** [comment]
This project will be around for the foreseeable future, but perhaps not this rule. If it continues to effect bugs, I will likely deprecate and delete it. There are other, more useful, rules in this package than the unsafe scope rule.

**Comment by dmytro-gokun** [comment]
Gotcha. Thanks for the explanation.
